hsmodels/schemas/base_models.py

```diff
@@ -100,7 +100,7 @@ def schema_extra(schema: Dict[str, Any], model) -> None:
                                 "title": "Key-Value",
                                 "description": "A key-value pair",
                                 "default": [],
-                                "properties": {"key": {"type": "string"}, "value": {"type": "string"}}
+                                "properties": {"key": {"type": "string"}, "value": {"type": "string"}},
```

Comment:
Should there be a comma at the end of this line?

Reply:
It was removed by the formatter.  It is the last item in the dictionary, the last comma is not necessary.